maths/modular_inverse.nim

```diff
@@ -0,0 +1,56 @@
+## Modular inverse
+
+import std/[options, math]
+
+import extended_gcd
```

Comment:
Should I change it to
```suggestion
import "extended_gcd.nim"
```
What is the desired way to import _own_ modules?

Reply:
https://nim-lang.org/docs/manual.html#modules-module-names-in-imports
In short:
- std for stdllib packages.
- pkg for external packages (dependencies only).
- ../module_name for modules in own repository.